puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/client/1.22.groovy

```diff
@@ -0,0 +1,75 @@
+pipeline {
+    agent none
+
+    environment {
+        version = "1.22"
+        branch = "rpm/${version}"
+    }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Mash Koji Repositories') {
+            agent { label 'sshkey' }
+
+            steps {
+                mash("foreman-client-mash-split-${version}.py")
```

Comment:
env.version ?

Reply:
Probably. It's currently a copy of what's in 1.21. Longer term I'd like to migrate to `mash("foreman-client-mash-split.py, env.version)` but I thought to leave that in a separate PR.